backend/src/server/createServer.ts

```diff
@@ -75,6 +75,13 @@ const createServer = async (
     logger,
   })
   apollo.applyMiddleware({ app, path: '/' })
+  logger.info(
+    `running in GRADIDO_ENV_NAME=${CONFIG.ENV_NAME} as PRODUCTION=${CONFIG.PRODUCTION} and EMAIL_TEST_MODUS=${CONFIG.EMAIL_TEST_MODUS} ...`,
+  )
+  if (CONFIG.PRODUCTION && CONFIG.EMAIL_TEST_MODUS === 'true') {
```

Comment:
Let us rely on the skills of the person doing the deploy

Reply:
the check is removed